doc/source/examples.rst

```diff
@@ -199,3 +199,24 @@ as the pull request gets updated with new commits.
           - base=master
         actions:
           dismiss_reviews:
+
+Require All Requested Reviews to Be Approved
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If all requested reviews have been approved, then the number of
+``review-requested`` and the number of ``changes-requested-reviews-by`` will
+both be 0.
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: remove outdated reviews
+        conditions:
+          - "#review-requested=0"
+          - "#changes-requested-reviews-by=0"
+        actions:
+            merge:
+              method: merge
+
+Note that if a requested review is dismissed, then it does count as a review
+that would prevent the merge.
```

Comment:
How's that? You mean `changes-requested-reviews-by` contains something?
That'd be weird.

Reply:
Awkward, that was a typo. It should say _doesn't_.